### PR TITLE
Issue#116 Refactor Action Bar

### PR DESCRIPTION
**Pull request by InnaAtanasova**
#### Please provide a link to the associated issue
[Issue#116](https://github.com/SAP/fundamental-ngx/issues/116)

#### Please provide a brief summary of this pull request
*****
`host: { class: 'some-class'}` allows the user to set up custom classes and does not overwrite them as in the case with dynamic classes - `host: { '[class]': 'some-class'}` 
*****
- adds back button `fd-action-bar-back`
- adds title and description for the Action Bar header
- removes the Action Bar input property for title (now title is a component).
- adds an `fd-action-bar-actions` component for the main actions. 
- adds a mobile view for the Action Bar `fd-action-bar-mobile`

### CODE REVIEW

File: projects/fundamental-ngx/src/lib/action-bar/action-bar-actions.component.ts [hunk 1]
```diff
@@ -0,0 +1,10 @@
+import { Component } from '@angular/core';
+
+@Component({
+    selector: 'fd-action-bar-actions',
+    host: {
+        class: 'fd-action-bar__actions'
```
Comment (droshev) [inline projects/fundamental-ngx/src/lib/action-bar/action-bar-actions.component.ts]: can we avoid this?

File: projects/fundamental-ngx/src/lib/action-bar/action-bar-back.component.ts [hunk 2]
```diff
@@ -0,0 +1,10 @@
+import { Component } from '@angular/core';
+
+@Component({
+    selector: 'fd-action-bar-back',
+    host: {
+        class: 'fd-action-bar__back'
```
Comment (droshev) [inline projects/fundamental-ngx/src/lib/action-bar/action-bar-back.component.ts]: can we avoid this?

File: projects/fundamental-ngx/src/lib/action-bar/action-bar-header.component.ts [hunk 3]
```diff
@@ -0,0 +1,10 @@
+import { Component } from '@angular/core';
+
+@Component({
+    selector: 'fd-action-bar-header',
+    host: {
```
Comment (droshev) [inline projects/fundamental-ngx/src/lib/action-bar/action-bar-header.component.ts]: can we avoid using host/class?

**Comment by InnaAtanasova** [comment]
host: { class: 'some-class'} allows the user to set up custom classes and does not overwrite them as in the case with dynamic classes - host: { '[class]': 'some-class'}

**Pull request merged**